index.js

```diff
@@ -177,6 +180,11 @@ function compression (options) {
       var negotiator = new Negotiator(req)
       var method = negotiator.encoding(['gzip', 'deflate', 'identity'], ['gzip'])
 
+      // if no method is found, use the default encoding
+      if (encodingSupported.indexOf(enforceEncoding) !== -1 && !req.headers['accept-encoding']) {
```

Comment:
nit: can you change the order of this? We should not check if the option is valid unless we are going to use it.

Reply:
sure